package.json

```diff
@@ -32,5 +33,8 @@
     "eslint-config-google": "^0.14.0",
     "eslint-plugin-cypress": "^2.12.1",
     "typescript": "^4.7.3"
+  },
+  "dependencies": {
+    "nvm": "^0.0.4"
```

Comment:
Why did you added this?

Reply:
to verify the configuration but it can be deleted

Reply:
I think its not necessary, we only require the .nvmrc file